grupo-gomez-lopez/src/App.js

```diff
@@ -1,24 +1,21 @@
-import logo from './logo.svg';
+import React from 'react';
 import './App.css';
+import { Route, Routes } from 'react-router-dom';
+import LoginView from './screems/LoginView/LoginView';
+import Layout from './screems/LayoutView/Layout.';
+import AboutView from './screems/AboutView/AboutView';
+import ProductsView from './screems/ProductsView/ProductsView';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React Grupo
-        </a>
-      </header>
-    </div>
+    <Layout>
+      <Routes>
+        <Route path='/' element={<LoginView />} />
+        <Route path='/about' element={<AboutView />} />
+        <Route path='/products' element={<ProductsView />} />
```

Comment:
Here you should be redirecting any other non-existent path of your web into your main screen

Reply:
ok, I understand.  We'll work in  that tomorrow.